From 490f85e73d8bca15ada1c0bacc101c136e5d9cee Mon Sep 17 00:00:00 2001
From: Cyrill Gorcunov <gorcunov@gmail.com>
Date: Thu, 27 Dec 2012 20:02:17 +0400
Subject: [PATCH] br3392236: Don't treat \Space after \BackSlash as a sign for line continuation

In commit f1fe4fdeabeaf2e5e4d02ef43beeb09a6fbfed1b I occasionally
made a \Space after \BackSlash being a sign of line continuation.

Fix it.

Signed-off-by: Cyrill Gorcunov <gorcunov@gmail.com>
---
 preproc.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/preproc.c b/preproc.c
index 8c5bf7a..9d65917 100644
--- a/preproc.c
+++ b/preproc.c
@@ -844,7 +844,7 @@ static char *read_line(void)
         case '\\':
             next = fgetc(istk->fp);
             ungetc(next, istk->fp);
-            if (next == ' ' || next == '\r' || next == '\n') {
+            if (next == '\r' || next == '\n') {
                 cont = true;
                 nr_cont++;
                 continue;
-- 
1.6.5.GIT

